java/test/jmri/RunCucumberIT.java

```diff
@@ -37,6 +37,15 @@ public static void beforeTests(){
    @AfterClass
    public static void afterTests(){
       jmri.util.web.BrowserFactory.CloseAllDriver();
+      if (InstanceManager.containsDefault(ShutDownManager.class)) {
```

Comment:
Why not use `JUnitUitl.clearShutdownManager()`?

Reply:
Because clear shutdown manager is deprecated and I don’t want this bit of code to be removed.

Because the cucumber tests are integration tests, the test runner doesn’t always know what is started.  ( i.e. we don’t create every object in test setup ) so it doesn’t know if a task is going to create a shutdown task or not.

This was prompted by a JSon object creating a shutdown task that stuck around and failed the test, despite the fact that all the test steps completed successfully.